update information-schema/information-schema-tiflash-replica.md (#10458)
pingcap/docs: information-schema/information-schema-tiflash-replica.md

+-----------------+-------------+------+------+---------+-------+
7 rows in set (0.01 sec)
```

> 

Reviewer: quick-start-with-htap.md L165
```
- `PROGRESS` means the progress of the replication. The value is between 0.0 and 1.0. 1 means at least one TiFlash replica is replicated.
```
[TiDB Cloud HTAP Quick Start](https://docs.pingcap.com/tidbcloud/tidb-cloud-htap-quickstart#step-3-compare-the-query-performance-between-row-based-storage-and-columnar-storage:~:text=PROGRESS%20means%20the%20progress%20of%20the%20replication.%20The%20value%20is%20between%200%20and%201.%201%20means%20at%20least%20one%20replica%20is%20replicated.) might need to be updated.

Author: Done.

Fields in the `TIFLASH_REPLICA` table are described as follows:

- `TABLE_SCHEMA`: the name of the database to which the table belongs.
- `TABLE_NAME`: the name of the table.
- `TABLE_ID`: the internal ID of the table, which is unique within a TiDB cluster.
- `REPLICA_COUNT`: the number of TiFlash replicas.
- `LOCATION_LABELS`: the LocationLabelList that is set when a TiFlash replica is created.

> - `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. The `AVAILABLE` does not change after it becomes `1` (available).

Author: ```suggestion
- `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. Once the value of this field becomes `1` (available), it will not change anymore.
```

> - `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. Once a replica becomes available, this status does not change any more.

Reviewer 2: ```suggestion
- `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. The `AVAILABLE` does not change after it becomes `1` (available).
```

> - `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. Once the value of this field becomes `1` (available), it will not change anymore.

Reviewer: ```suggestion
- `AVAILABLE`: indicates whether the TiFlash replica of the table is available. When the value is `1` (available), the TiDB optimizer can intelligently choose to push down queries to TiKV or TiFlash based on query cost. When the value is `0` (unavailable), TiDB will not push down queries to TiFlash. Once the value of this field becomes `1` (available), it will not change anymore.
```

> - `PROGRESS`: the replication progress of TiFlash replicas, accurate to two decimal places and refreshed at a minute level. The scope of this field is `[0, 1]`. When the `AVAILABLE` field is `1` and `PROGRESS` is less than 1, the TiFlash replica is far behind TiKV, and the queries pushed down to TiFlash will probably fail due to timeout of waiting for data replication.

Reviewer: ```suggestion
- `PROGRESS`: the replication progress of TiFlash replicas, accurate to two decimal places and refreshed every minute. The scope of this field is `[0, 1]`. When the `AVAILABLE` field is `1` and `PROGRESS` is less than 1, the TiFlash replica is far behind TiKV, and the queries pushed down to TiFlash will probably fail due to timeout of waiting for data replication.
```

> - `PROGRESS`: the replication progress of TiFlash replicas, accurate to two decimal places and refreshed every minute. The scope of this field is `[0, 1]`. When the `AVAILABLE` field is `1` and `PROGRESS` is less than 1, the TiFlash replica is far behind TiKV, and the queries pushed down to TiFlash will probably fail due to timeout of waiting for data replication.

Reviewer 2: > refreshed every minute

the accuracy is at the minute level

> - `PROGRESS`: the replication progress of TiFlash replicas, accurate to two decimal places and refreshed every minute. The scope of this field is `[0, 1]`. When the `AVAILABLE` field is `1` and `PROGRESS` is less than 1, the TiFlash replica is far behind TiKV, and the queries pushed down to TiFlash will probably fail due to timeout of waiting for data replication.

Author: ```suggestion
- `PROGRESS`: the replication progress of TiFlash replicas, accurate to two decimal places and refreshed at the minute level. The scope of this field is `[0, 1]`. When the `AVAILABLE` field is `1` and `PROGRESS` is less than 1, the TiFlash replica is far behind TiKV, and the queries pushed down to TiFlash will probably fail due to timeout of waiting for data replication.
```
